src/adc_sample_conv.hpp

```diff
@@ -67,7 +67,7 @@ namespace adiscope {
 		void setHardwareGain(int connection, float gain);
 		float hardwareGain(int connection) const;
 
-		int work(int noutput_items,
+		int work(unsigned int noutput_items,
```

Comment:
This method here overrides work in [sync_block.h](https://github.com/gnuradio/gnuradio/blob/master/gnuradio-runtime/include/gnuradio/sync_block.h#L52) which is a pure virtual method, changing the signature of this method will cause errors.